Generators/include/Generators/GeneratorPythia8.h

```diff
@@ -54,8 +54,8 @@ class GeneratorPythia8 : public Generator
   GeneratorPythia8& operator=(const GeneratorPythia8&);
 
   /** methods to override **/
-  Bool_t generateEvent() override;
-  Bool_t importParticles() override;
+  virtual Bool_t generateEvent() override;
+  virtual Bool_t importParticles() override;
```

Comment:
`override` signifies that these functions are already virtual. The modern way is not to put virtual ... which is only done in the base class.

Reply:
`virtual` removed.